### ISSUE TITLE
error message using R 4.3.1

### ISSUE THREAD

**Issue by tolitsmagadia** [kolaczyk/sand#12]
Third command line in Chapter 1 gives the following:

> install_sand_packages()
Error: With R version 3.5 or greater, install Bioconductor packages using BiocManager; see https://bioconductor.org/install

I have installed latest version of BiocManager but don't know which Bioconductor packages are needed to run the rest of the chapters although those that depend on igraph work perfectly fine.

**Comment by gaborcsardi** [comment]
Unfortunately that function does not work anyt more. You can install dependencies like this:
```
pkgs <- strsplit(packageDescription("sand")$Suggests, "[, \n]+")[[1]]
# networkTomography was archived and does not build on newer R
pkgs <- setdiff(pkgs, "networkTomography")
pak::pkg_install(pkgs)
```

**Comment by tolitsmagadia** [comment]
Installing...

             
ℹ Building GenomeInfoDbData 1.2.10
Installing...

             
Error: 
! error in pak subprocess
Caused by error: 
! Could not find tools necessary to compile a package
Call `pkgbuild::check_build_tools(debug = TRUE)` to diagnose the problem.
Type .Last.error to see the more details.

**Comment by gaborcsardi** [comment]
Yes, to build R packages from source you need to install Rtools on Windows, and the Xcode command line tools on macOS.
